src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -614,7 +657,7 @@ void activity_classifier::init_train(
 }
 
 void activity_classifier::perform_training_iteration() {
-
+  
```

Comment:
Let's avoid introducing trailing whitespace (the two spaces on this line with no text after it)